Doc/library/string.rst

```diff
@@ -755,8 +755,18 @@ attributes:
 
 * *idpattern* -- This is the regular expression describing the pattern for
   non-braced placeholders.  The default value is the regular expression
-  ``[_a-z][_a-z0-9]*``.  If this is given and *braceidpattern* is ``None``
-  this pattern will also apply to braced placeholders.
+  ``(?-i:[_a-zA-Z][_a-zA-Z0-9]*)``. Since default *flags* is
+  ``re.IGNORECASE``, ``[a-z]``Without local flag ``-i``, is used to avoid to match with non ASCII characters.
```

Comment:
There should be a space before the `W`, but I also feel like the sentence starting with "Since default..." should just be moved to, and consolidated with, the `note::` section that just follows.

Reply:
Oh, I missed removing this sentence after writing note section.